core/api/src/servers/graphql-server.ts

```diff
@@ -46,40 +48,34 @@ export const startApolloServer = async ({
   const httpServer = createServer(app)
 
   const apolloPlugins = [
-    createComplexityPlugin({
-      schema,
-      estimators: [fieldExtensionsEstimator(), simpleEstimator({ defaultComplexity: 1 })],
-      maximumComplexity: 200,
-      onComplete: (complexity) => {
-        // TODO(telemetry): add complexity value to span
-        baseLogger.debug({ complexity }, "queryComplexity")
-      },
-    }),
+    // createComplexityPlugin({
```

Comment:
this doesn't work with apollo 4?

Reply:
maybe we should just import the plugin to our code, this gist is very small: https://gist.github.com/reconbot/37ccd8b8fcdfb78ff6a16a7055edf103

Reply:
I used the code from npm source.. it was a bit different from gist